src/react/components/pages/export/exportForm.tsx

```diff
@@ -160,8 +159,9 @@ export default class ExportForm extends React.Component<IExportFormProps, IExpor
 
         const formData = { ...exportFormat };
         if (resetProviderOptions) {
-            formData.providerOptions = { assetState: ExportAssetState.All };
+            formData.providerOptions = {} as IExportProviderOptions;
```

Comment:
clear out previous settings and let the form component pull default from the export provider json file

Reply:
Does this work as expected to respect the default values?

Reply:
> Does this work as expected to respect the default values?

yup it does :D